docs/developing_in_force.md

```diff
@@ -0,0 +1,38 @@
+# Developing in force
+
+Often in development you'll want to link the development version of reaction into force.
+Traditionally one would use `yarn link` in reaction and `yarn link @artsy/reaction` in a service like force
+to achieve this. Unfortunately, this mechanism is prone to a lot of issues. Below is a guide for a smoother
```

Comment:
Doesn't have to be too verbose, but could you provide some color as to what the issues are?